src/portfolios/portfolio/components/TaskOrder/TaskOrderDetails.spec.ts

```diff
@@ -132,19 +133,18 @@ describe("Testing TaskOrderDetails Component", () => {
       localVue,
       vuetify,
       propsData: {
-        selectedTaskOrder: [
+        selectedTaskOrder:
           {
```

Comment:
- [ ] changed from an array to object.  Will it cause any problems?

Reply:
the `selectedTaskOrder` was always defined as type `TaskOrderCardData` not an array of `TaskOrderCardData` I talked with @DGabriel-CCPO extensively about this as I was trying to get this test file to run properly.